### DISCUSSION TITLE
Use directus app non-global components in extension

### DISCUSSION THREAD

**Discussion by skopz356** [directus/directus#11747] [Archive]
Hi, 
is there an option to use directus non-global components in extension. 
By non-global components I mean components such as DrawerItem or DrawerCollection or option to import anything from directus/app.

Thank you for your great work and have a nice day!

**Comment by joselcvarela** [comment]
Hello,
I believe that is not possible since those components are tied with stores.
But you can always copy/paste the code and adapt to your needs 😅

**Reply by skopz356** [reply]
Okay thanks, lets copy/paste :smile:
